src/components/rectification/RectificationForm.tsx

```diff
@@ -0,0 +1,202 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Checkbox,
+  IconCheckCircle,
+  Link,
+  RadioButton,
+  Select,
+  SelectionGroup,
+  TextArea,
+  TextInput
+} from 'hds-react';
+import { useClient } from '../../client/hooks';
+
+import './RectificationForm.css';
+
+const RectificationForm = () => {
+  const { t } = useTranslation();
+
+  const { getUser } = useClient();
+  const user = getUser();
+
+  const [checked, setChecked] = useState(false);
+  const [newEmailSelected, setNewEmailSelected] = useState(false);
+  const [vehicleRelation, setVehicleRelation] = useState('driver');
+  const [currentCharacters, setCurrentCharacters] = useState(0);
+
+  const [decision, setDecision] = useState('toParkingService');
+
+  const handleCheckbox = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setChecked(e.target.checked);
+    setNewEmailSelected(e.target.checked);
+  };
+
+  const handleVehicleRelation = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setVehicleRelation(e.target.value);
+  };
+
+  const handleDecision = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setDecision(e.target.value);
+  };
+
+  const relations = ['driver', 'owner', 'holder'];
+
+  return (
+    <>
+      <p>{t('common:required-fields')}</p>
+      <div className="userDetails">
+        <SelectionGroup label={t('rectification:relation')}>
+          {relations.map(relation => (
+            <RadioButton
+              key={relation}
+              label={t(`rectification:${relation}`)}
+              id={relation}
+              value={relation}
+              checked={vehicleRelation === relation}
+              onChange={handleVehicleRelation}
+            />
+          ))}
+        </SelectionGroup>
+
+        <div className="rectification-form-user-details">
+          <TextInput
+            id="fullName"
+            readOnly
+            label={t('common:name')}
+            defaultValue={user?.name as string}
+          />
+          <IconCheckCircle aria-label={t('common:fetched-from-profile-aria')} />
```

Comment:
Linter doesn't like this line being repeated 3 times:

_Define a constant instead of duplicating this literal 3 times  sonarjs/no-duplicate-string_

Reply:
Suppressed the warning since there isn't really a good case for defining a constant for a label text